### PR TITLE
add github actions

### PR DESCRIPTION
**Pull request by Felixmil**

### CODE REVIEW

**Comment by rwmcintosh** [comment]
Fixes #128

**Comment by Felixmil** [comment]
- [x] R CMD check on windows and ubuntu (3 r versions are tested on ubuntu: devel, latest released and previous released)
- [x] Pkgdown with automatic deployement to gh-pages
- [x] testcoverage

File: shared/rSharp.h [hunk 1]
```diff
@@ -124,6 +124,8 @@ extern "C" {
 	SEXP r_get_typename_externalptr(SEXP parameters);
 	SEXP make_char_single_sexp(const char* str);
 	SEXP rsharp_object_to_SEXP(RSharpGenericValue& objptr);
+	SEXP r_get_sexp_type(SEXP par);
```
Comment (PavelBal) [inline shared/rSharp.h:127]: Why, where does this change come from?

@rwmcintosh do you know?

**Review by PavelBal — CHANGES_REQUESTED** [review]

**Review by PavelBal — APPROVED** [review]
Pure awesomeness

**Pull request merged**